core/src/alpenglow_consensus/voting_loop.rs

```diff
@@ -60,6 +63,8 @@ use {
     },
 };
 
+type PendingBlocks = BTreeMap<Slot, Arc<Bank>>;
```

Comment:
Would be helpful to add a comment here indicating these are blocks that have completed replay but need to be voted on